src/nrepl/config.clje

```diff
@@ -46,12 +49,13 @@
       {})))
 
 ;; MB: if this is called too often we should consider putting this into an Atom.
-(defn- config
+(defn- config []
```

Comment:
I think this `[]` here is not necessary.

Reply:
That's right.
I'm not sure this actually produces a runtime error.
But I'll try add some test cases for this now anyway, now that it compiles.